RestaurantApp.Android/RestaurantApp.Android.csproj

```diff
@@ -55,6 +55,9 @@
     <PackageReference Include="Xamarin.Forms" Version="4.2.0.709249" />
     <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.1" />
     <PackageReference Include="Xamarin.Essentials" Version="1.2.0" />
+    <PackageReference Include="sqlite-net-pcl">
+      <Version>1.6.292</Version>
+    </PackageReference>
   </ItemGroup>
```

Comment:
do not forgot to add all packages to all platforms (arc user dialog is missing here)